terraform/modules/ignition/ignition.tf

```
@@ -14,45 +14,47 @@
 
 data "ignition_config" "main" {
   files = [
-    "${data.ignition_file.eco-config.id}",
-    "${data.ignition_file.eco-ca.id}",
-    "${data.ignition_file.eco-crt.id}",
-    "${data.ignition_file.eco-key.id}",
-    "${data.ignition_file.eco-health.id}",
-    "${data.ignition_file.e.id}",
+    data.ignition_file.eco-config.id,
+    data.ignition_file.eco-ca.id,
+    data.ignition_file.eco-crt.id,
+    data.ignition_file.eco-key.id,
+    data.ignition_file.eco-health.id,
+    data.ignition_file.e.id,
   ]
 
   systemd = [
-    "${data.ignition_systemd_unit.docker.id}",
-    "${data.ignition_systemd_unit.locksmithd.id}",
-    "${data.ignition_systemd_unit.update-engine.id}",
-    "${data.ignition_systemd_unit.eco.id}",
-    "${data.ignition_systemd_unit.eco-health.id}",
-    "${data.ignition_systemd_unit.node-exporter.id}",
+    data.ignition_systemd_unit.docker.id,
+    data.ignition_systemd_unit.locksmithd.id,
+    data.ignition_systemd_unit.update-engine.id,
+    data.ignition_systemd_unit.eco.id,
+    data.ignition_systemd_unit.eco-health.id,
+    data.ignition_systemd_unit.node-exporter.id,
   ]
 
-  users = ["${data.ignition_user.core.id}"]
+  users = [data.ignition_user.core.id]
 
   append {
-    source       = "${lookup(var.ignition_extra_config, "source", local.blank_ignition_config)}"
-    verification = "${lookup(var.ignition_extra_config, "verification", "")}"
+    source = lookup(
+      var.ignition_extra_config,
+      "source",
+      local.blank_ignition_config,
+    )
```

nit: this lookup got split in several lines, while the other stayed a one liner

---

Oh -- sorry. In fact my "coding style" would have been by default to break both up, as I like to have multiple "strips" of code open in different vertical panes. 
```suggestion
    source = lookup(var.ignition_extra_config, "source", local.blank_ignition_config)
```